src/utils/settingsService.js

```diff
@@ -6,3 +6,7 @@ export const getSetting = (key, defaultValue) => {
 export const setSetting = (key, value) => {
   localStorage.setItem(key, JSON.stringify(value));
 };
+
+export function capitalizeFirstLetter(str) {
+  return str.charAt(0).toUpperCase() + str.substring(1);
```

Comment:
we can capitalise the first letter  through css also so if we only want to show we can use 
```css
text-transform: capitalize;
```